src/Generators/AbstractClassGenerator.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Blueprint\Generators;
+
+use Blueprint\Blueprint;
+use Blueprint\Concerns\HandlesImports;
+use Blueprint\Concerns\HandlesTraits;
+use Blueprint\Contracts\Model;
+use Illuminate\Filesystem\Filesystem;
+
+class AbstractClassGenerator
+{
+    use HandlesImports, HandlesTraits;
```

Comment:
Is it true that all subclasses handle imports? Same for traits?

Reply:
It's not true that all subclasses handle imports and traits. It is currently Controller/Seeder/Factory/Test Generators (x4) that make use of the traits, and potentially the 5th with a view to the Model generator starting to use it. So with 5 classes, its around +20 lines (4 per class) and -4 lines from the abstract class.

I was going with the view that, all class generators should have the 'ability' to handle imports and traits, but I take your point, given that Migration, Route and Statement generators don't use this.

I have amended the pull request, along with the other property suggestions